lib/site_prism/section.rb

```diff
@@ -9,10 +9,10 @@ class Section
 
     attr_reader :root_element, :parent
 
-    def initialize(parent, root_element)
+    def initialize(parent, root_element, &block)
       @parent = parent
       @root_element = root_element
-      Capybara.within(@root_element) { yield(self) } if block_given?
+      within(&block) if block_given?
```

Comment:
looks like the rubocop fixes were made on top of your other branch/PR, you should cherry pick the rubocop fixes onto master or include them on your branch, depending on @natritmeyer's preference.